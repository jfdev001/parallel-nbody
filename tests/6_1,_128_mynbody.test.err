Reservation number 3909814: Reserved 6 hosts for 900 seconds 
Run on 6 hosts for 960 seconds from Sun Jan 15 15:37:21 2023
: node059/0 node060/0 node061/0 node062/0 node063/0 node064/0 
Running N-body with 128 bodies and 1000 steps
