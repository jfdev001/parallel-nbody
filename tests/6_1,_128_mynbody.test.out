[node059:31372] *** An error occurred in MPI_Gather
[node059:31372] *** reported by process [1189216257,0]
[node059:31372] *** on communicator MPI_COMM_WORLD
[node059:31372] *** MPI_ERR_TRUNCATE: message truncated
[node059:31372] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
[node059:31372] ***    and potentially your MPI job)
STATUS=PASSED
